coder_sniffer/Drupal/Sniffs/Commenting/DeprecatedSniff.php

```diff
@@ -198,19 +198,24 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
-        // Check the format of the @see url.
         $string = $phpcsFile->findNext(T_DOC_COMMENT_STRING, ($seeTag + 1), $commentEnd);
-        $crLink = $tokens[$string]['content'];
-        // Allow for the alternative 'node' or 'project/aaa/issues' format.
-        preg_match('[^http(s*)://www.drupal.org/(node|project/\w+/issues)/(\d+)(\.*)$]', $crLink, $matches);
-        if (isset($matches[4]) === true && empty($matches[4]) === false) {
-            // If matches[4] is not blank it means that the url is correct but
-            // it ends with a period. This is covered by the generic fixable
-            // sniff FunctionCommentSniff.SeePunctuation so allow that sniff to
-            // report it (and fix it).
-        } else if (empty($matches) === true) {
-            $error = "The @see url '%s' does not match the standard: http(s)://www.drupal.org/node/n or http(s)://www.drupal.org/project/aaa/issues/n";
-            $phpcsFile->addWarning($error, $seeTag, 'DeprecatedWrongSeeUrlFormat', [$crLink]);
+        // If the @see tag exists but has no content then do nothing more as
+        // this is covered by FunctionComment.EmptySees sniff. It also causes
+        // an odd message with '<?php' shown for the @see url content.
+        if (!empty($string)) {
+            // Check the format of the @see url.
```

Comment:
This is just a simple change, wrapping the url check in a conditional, to avoid doing anything when there is nothing to check. This is covered by FunctionComment.EmptySees anyway.

Reply:
Backed out this change, as the other sniff is only for functions, so was not always being shown. Now cater for blank @see url by setting $crLink to blank. Two commits squashed together for ease of review.